quickstart/.travis.yml

```diff
@@ -9,5 +9,4 @@ tex-config:
 - packages=lipsum, blindtext
 
 script:
-- docker pull strauman/travis-latexbuild:small # select small texlive scheme
-- docker run --mount src=$TRAVIS_BUILD_DIR/,target=/repo,type=bind strauman/travis-latexbuild:small # also it here
\ No newline at end of file
+- docker run --mount src=$TRAVIS_BUILD_DIR/,target=/repo,type=bind strauman/travis-latexbuild:small # Select small TeX Live scheme
```

Comment:
Should it maybe be selected automatically somehow? I'll make this a separate issue at some point.

Reply:
Oh, you mean because it's already specified in the options? Agree, would be an enhancement.